signer/signer_trust.go

```diff
@@ -30,10 +31,14 @@ type NotarySigner struct {
 func NewNotarySigner(hostname string, port string, tlscafile string) *NotarySigner {
 	var opts []grpc.DialOption
 	netAddr := net.JoinHostPort(hostname, port)
-	creds, err := credentials.NewClientTLSFromFile(tlscafile, hostname)
+	tlsConfig, err := utils.ConfigureClientTLS(&utils.ClientTLSOpts{
+		RootCAFile: tlscafile,
```

Comment:
I have updated `ConfigureClientTLS` to use `CertPool.AppendCertsFromPEM` in 09dc607, so hopefully that addresses this issue.  :)